app/server/lib/UpdateManager.ts

```diff
@@ -38,6 +37,31 @@ export const Deps = {
   BAD_RESULT_TTL,
 };
 
+/**
+ * JSON returned to the client (exported for tests).
+ */
+export interface LatestVersion {
+  /**
+   * Latest version of core component of the client.
+   */
+  latestVersion: string;
+  /**
+   * If there were any critical updates after client's version. Undefined if
+   * we don't know client version or couldn't figure this out for some other reason.
+   */
+  isCritical?: boolean;
```

Comment:
@jordigh Do you know if there's any existing styling or behavior change based on this flag? We'll be able to upgrade our side to set `isCritical` when needed, but we won't be able to upgrade user installations to take special action on it except with the slow passage of time. If there were some easy way to make critical updates very prominent, it would make this feature more valuable to administrators earlier.

Reply:
A clear visual difference such as color, combined with some text about urgency, would be useful and suffice for now.